src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
Ah yes but I think that it is orthogonal to this decision.

Using the current pattern the yaml would look like.

```
language_settings:
  python:
    package_name: google.cloud.example_v1
    gapic_package_name: google.cloud.gapic.example.v1
    proto_package_name: google.cloud.proto.example.v1